robottelo/content_info.py

```diff
@@ -0,0 +1,77 @@
+"""Miscellaneous content helper functions"""
+import re
+
+import requests
+
+from robottelo import ssh
+from robottelo.cli.base import CLIReturnCodeError
+
+
+def get_repo_files(repo_path, extension='rpm', hostname=None):
+    """Returns a list of repo files (for example rpms) in specific repository
+    directory.
+
+    :param str repo_path: unix path to the repo, e.g. '/var/lib/pulp/fooRepo/'
+    :param str extension: extension of searched files. Defaults to 'rpm'
+    :param str optional hostname: hostname or IP address of the remote host. If
+        ``None`` the hostname will be get from ``main.server.hostname`` config.
+    :return: list representing rpm package names
+    :rtype: list
+    """
+    if not repo_path.endswith('/'):
+        repo_path += '/'
+    result = ssh.command(
+        f"find {repo_path} -name '*.{extension}' | awk -F/ '{{print $NF}}'",
+        hostname=hostname,
+    )
```

Comment:
Optional: Why keeping `ssh.command` here? can't we move these methods under ContentHost class?

Reply:
that's something that might happen in a later helpers refactor, but not something that is needed just yet